### PR TITLE
Hightouch Cohort Import Docs - MERGE 5/9

### PR DESCRIPTION
**Pull request by KellieHawks**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the interal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@Timothy-Kim</td>
    <td>Data Infrastructure<br>Platform Infrastructure<br>Quality Infrastructure</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Internal Tools<br>Product Partnerships<br>SDKs<br>SMS</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Channels<br>FIX<br>Intelligence<br>Reporting<br>SMB</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Email (Composition and Infrastructure)<br>Ingestion<br>Messaging</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_partners/data_and_infrastructure_agility/workflow_automation/hightouch.md [hunk 1]
```diff
@@ -52,14 +55,35 @@ You can sync Hightouch objects (user fields) to the equivalent Braze default or
...
 Further instructions on object and event syncing can be found in [Hightouch documentation](https://hightouch.io/docs/destinations/braze/).
 {% endalert %}
 
+## Data import integration
+
+### Step 1: Get the Braze data import Key
+In Braze, navigate to **Technology Partners** and select **Hightouch**. Here, you will find your REST Endpoint and generate your Braze data import key. Once generated, you can create a new key or invalidate an existing one.<br><br>![][6]{: style="max-width:90%;"} 
+
+### Step 2: Add Braze Cohorts as a Destination in Hightouch
+Navigate to the **Destination** page in your Hightouch workspace, search for **Braze Cohorts** and click **Continue**. From there, take your REST endpoint and data import key and click **Continue**.<br><br>![][7]{: style="max-width:90%;"}
```
Comment (lydia-xie) [inline _docs/_partners/data_and_infrastructure_agility/workflow_automation/hightouch.md:70]:
```suggestion
Navigate to the **Destination** page in your Hightouch workspace, search for **Braze Cohorts**, and click **Continue**. From there, take your REST endpoint and data import key and click **Continue**.<br><br>![][7]{: style="max-width:90%;"}
```

File: _docs/_partners/data_and_infrastructure_agility/workflow_automation/hightouch.md [hunk 2]
```diff
@@ -52,14 +55,35 @@ You can sync Hightouch objects (user fields) to the equivalent Braze default or
...
+
+### Step 3: Sync a model (or audience) into Braze Cohorts
+In Hightouch, using a [model](https://hightouch.io/docs/getting-started/create-your-first-sync/#create-a-model) or [audience](https://hightouch.io/docs/audiences/usage/) you created and create a new sync. Nest, select the Braze Cohorts destination you created in the previous step. Lastly, in the Braze Cohorts destination configuration, select the identifier you want to match against and decide whether or not you want Hightouch to create a new Braze Cohort or update an existing one.<br><br>![][8]{: style="max-width:90%;"}
```
Comment (lydia-xie) [inline _docs/_partners/data_and_infrastructure_agility/workflow_automation/hightouch.md:73]:
```suggestion
In Hightouch, using your created [model](https://hightouch.io/docs/getting-started/create-your-first-sync/#create-a-model) or [audience](https://hightouch.io/docs/audiences/usage/), create a new sync. Next, select the Braze Cohorts destination you created in the previous step. Lastly, in the Braze Cohorts destination configuration, select the identifier you want to match against and decide whether or not you want Hightouch to create a new Braze Cohort or update an existing one.<br><br>![][8]{: style="max-width:90%;"}
```
Wondering on the capitalization of "Braze Cohorts" here vs. "Hightouch cohort" in Step 4 and if it should be aligned? 🤔

**Review by lydia-xie — CHANGES_REQUESTED** [review]
Looks great! Left a couple of suggestions for your review!

File: _docs/_partners/data_and_infrastructure_agility/workflow_automation/hightouch.md [hunk 3]
```diff
@@ -52,14 +55,35 @@ You can sync Hightouch objects (user fields) to the equivalent Braze default or
...
 {% alert important %}
 Further instructions on object and event syncing can be found in [Hightouch documentation](https://hightouch.io/docs/destinations/braze/).
 {% endalert %}
 
+## Data import integration
+
+### Step 1: Get the Braze data import Key
+In Braze, navigate to **Technology Partners** and select **Hightouch**. Here, you will find your REST Endpoint and generate your Braze data import key. Once generated, you can create a new key or invalidate an existing one.<br><br>![][6]{: style="max-width:90%;"} 
+
+### Step 2: Add Braze Cohorts as a Destination in Hightouch
```
Comment (KellieHawks) [inline _docs/_partners/data_and_infrastructure_agility/workflow_automation/hightouch.md:69]:
```suggestion
### Step 2: Add Braze cohorts as a Destination in Hightouch
```

File: _docs/_partners/data_and_infrastructure_agility/workflow_automation/hightouch.md (hunk 2 above)
Comment (KellieHawks) [reply _docs/_partners/data_and_infrastructure_agility/workflow_automation/hightouch.md]: Good call! Updated!

**Review by lydia-xie — APPROVED** [review]

**Pull request merged**